build_projects/dotnet-host-build/PublishTargets.cs

```diff
@@ -230,6 +230,7 @@ private static bool CheckIfAllBuildsHavePublished()
             nameof(PublishTargets.PublishDotnetDebToolPackage),
             nameof(PublishTargets.PublishDebFilesToDebianRepo),
             nameof(PublishTargets.PublishCoreHostPackages),
+            nameof(PublishTargets.PublishSignedPackages),
```

Comment:
I think identifying what's being published is more important than identifying the reason - I'll change this